test/ex_doc/formatter/html/autolink_test.exs

```diff
@@ -393,7 +398,11 @@ defmodule ExDoc.Formatter.HTML.AutolinkTest do
     end
 
     test "autolinks inside parameterized types" do
-      assert Autolink.typespec(quote(do: parameterized_t(foo())), [parameterized_t: 1, foo: 0], []) ==
+      assert Autolink.typespec(
+               quote(do: parameterized_t(foo())),
+               [parameterized_t: 1, foo: 0],
+               []
```

Comment:
Can we move at least one of those to a var so the call fits a single line?

Reply:
Would shortening the type name be a good compromise? For example `p_t(foo())`? The test description should be enough to learn that these are parameterized.

Reply:
sounds good to me.